src/style.css

```
@@ -0,0 +1,107 @@
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 800;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  --main-color1: #e1e7ed;
+  --main-color2: #3d4e61;
+}
+
+.body {
+  margin: 0;
+}
+
+.form {
+  justify-content: center;
+  margin: 20px 0;
+}
+
+.search_label {
+  position: relative;
+  display: block;
+  width: 100%;
+}
+
+.search_label:after {
+  position: absolute;
+  content: "";
+  display: block;
+}
+
+.search_label_big:after {
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
```

Combine rules into background and replace! to search_label:after not to be repeated
```suggestion
  background-image: url(images/Search.svg);
  background-repeat: no-repeat;
  background-size: contain;
```
```suggestion
```